src/main/resources/assets/minestuck/models/block/loot_chest.json

```diff
@@ -0,0 +1,137 @@
+{
+  "credit": "Made with Blockbench",
+  "texture_size": [64, 64],
+  "textures": {
+    "0": "minestuck:block/chest_texture",
+    "particle": "minestuck:block/chest_texture"
+  },
+  "display": {
+    "thirdperson_righthand": {
+      "rotation": [75, 45, 0],
+      "translation": [0, 2.5, 0],
+      "scale": [0.375, 0.375, 0.375]
+    },
+    "thirdperson_lefthand": {
+      "rotation": [75, 45, 0],
+      "translation": [0, 2.5, 0],
+      "scale": [0.375, 0.375, 0.375]
+    },
+    "firstperson_righthand": {
+      "rotation": [0, 45, 0],
+      "scale": [0.4, 0.4, 0.4]
+    },
+    "firstperson_lefthand": {
+      "rotation": [0, 225, 0],
+      "scale": [0.4, 0.4, 0.4]
+    },
+    "ground": {
+      "translation": [0, 3, 0],
+      "scale": [0.25, 0.25, 0.25]
+    },
+    "gui": {
+      "rotation": [30, 225, 0],
+      "scale": [0.625, 0.625, 0.625]
+    },
+    "fixed": {
+      "scale": [0.5, 0.5, 0.5]
+    }
+  },
+  "elements": [
+    {
+      "from": [0, 0, 0],
+      "to": [16, 9, 15],
+      "faces": {
+        "north": {"uv": [0, 8, 4, 10.25], "texture": "#0"},
+        "east": {"uv": [4, 0, 7.75, 2.25], "texture": "#0"},
+        "south": {"uv": [0, 0, 4, 2.25], "texture": "#0"},
+        "west": {"uv": [4, 0, 7.75, 2.25], "texture": "#0"},
+        "up": {"uv": [4, 4.5, 0, 0.5], "texture": "#0"},
+        "down": {"uv": [4, 3, 0, 7], "texture": "#0"}

```

Comment:
The 16x16 texture for the up and down sides does not quite match up with the 16x15 size of those sides

Reply:
There are probably other petty texture things elsewhere, but I will not go searching for them now